PipelinesTasks/PRMetrics/buildAndReleaseTask/.vscode/launch.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
This doesn't meet the editorconfig rules. I'd suggest installing the editorconfig extension for VS Code which will keep things in line.

As is, this will break the build pipe